src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -101,6 +107,8 @@ export class BasicAutocompleteComponent<O, V = O>
    * Whether the user should be able to select multiple values.
    */
   @Input() multi?: boolean;
+  @Input() reorder?: boolean;
+  autocompleteDraggableOptions: SelectableOption<O, V>[] = [];
```

Comment:
can we not use the same autocompleteSuggestedOptions array also if drag&drop is availalbe?

Reply:
The drop method expects an array as input to perform operations such as moving items within the array. However, autocompleteSuggestedOptions is an Observable that emits values asynchronously, and it's not directly usable as an array in the drop method. That is why I am using different arrays to drag and drop.

Reply:
The drop method expects an array as input to perform operations such as moving items within the array. However, autocompleteSuggestedOptions is an Observable that emits values asynchronously, and it's not directly usable as an array in the drop method. That is why I am using different arrays to drag and drop.